PcBdsPkg/Include/Library/MsBootPolicyLib.h

```diff
@@ -22,6 +22,8 @@ typedef enum {
   MsBootDone,       /// Boot Sequence terminator, used to exit boot application
   MsBootPXE4,       /// Boot devices that support IPV4 PXE
   MsBootPXE6,       /// Boot devices supporting IPV6 PXE
+  MsBootHttp4,       /// Boot devices that support IPV4 Http

```

Comment:
This is an enum. If you are adding new entries, they should be added at the end of existing entries.

Reply:
done.